src/display/api.js

```diff
@@ -755,22 +757,8 @@ class PDFDocumentProxy {
    */
 
   /**
-   * @returns {Promise<Array<OutlineNode>>} A promise that is resolved with
-   * an {Array} that is a tree outline (if it has one) of the PDF. The tree is
-   * in the format of:
-   * [
-   *   {
-   *     title: string,
-   *     bold: boolean,
-   *     italic: boolean,
-   *     color: rgb Uint8ClampedArray,
-   *     count: integer or undefined,
-   *     dest: dest obj,
-   *     url: string,
```

Comment:
This is listed as `any` in the typedef, when it probably ought to be `<string | null>`.

---

I know that you didn't write it, but please double-check the `typedef` to ensure that it makes sense (in addition to the things I spotted above).